packages/paste-website/src/pages/getting-started/inclusive-design/index.mdx

```diff
@@ -0,0 +1,285 @@
+
+**Keep us accountable:** Track our progress on our [GitHub Discussions thread](https://github.com/twilio-labs/paste/discussions/561) as we apply these principles across this website.
+
+## How to use this guide
+
+We recommend using this guide in the following ways:
+
+- Read through the guide once. We recommend starting with sections that relate to your role. However, reading the entire list will give you a richer understanding of inclusive design and accessibility.
+- Share this guide with your design, product, and engineering peers. Engage with them in a conversation about how we could cave some space in our day-to-day work to make our Twilio experiences a little more inclusive.
+- Study the guide at natural checkpoints of your product's design and development: we recommend doing so at the start of a project, before a UX review, and as a pre-launch checklist.
+- This is not a comprehensive list. Product design and UX engineering are complex functions that can't be solved with a checklist. Instead, consider this as a way to spot areas of opportunity for your product.
+
+## Inclusive design considerations
+
+<Stack orientation="vertical" spacing="space50">
+  <Disclosure visible variant="contained">
+  <DisclosureHeading as="h3" variant="heading30">
+    General
+  </DisclosureHeading>
+  <DisclosureContent>
+    <UnorderedList>
+      <ListItem>Remove potentially insensitive or uninclusive language, and use "singular they" pronouns in UIs. <Anchor href="http://alexjs.com/">Consider using a service like alexjs to catch any issues.</Anchor></ListItem>
+      <ListItem>Give video content captions and transcripts</ListItem>
+      <ListItem>Provide transcripts for audio content</ListItem>
+      <ListItem>Make sure controls do not elicit unexpected or jarring behavior (For example, do not change behavior or context unexpectedly when a user <Anchor href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-receive-focus.html">changes focus</Anchor> or <Anchor href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html">adjusts a control</Anchor></ListItem>
+      <ListItem>Provide alternative text for salient images <Anchor href="https://www.w3.org/WAI/tutorials/images/decision-tree/">(Why it matters)</Anchor></ListItem>
+      <ListItem>Make sure text and background colors contrast sufficiently <Anchor href="https://accessibility.blog.gov.uk/2016/06/17/colour-contrast-why-does-it-matter/">(Why it matters)</Anchor></ListItem>
+              <ListItem>Ensure PDF content is accessible (include tags) <Anchor href="https://webaim.org/techniques/acrobat/">(Why it matters)</Anchor></ListItem>
+    </UnorderedList>
+  </DisclosureContent>
+</Disclosure>
+
+<Disclosure variant="contained">
+  <DisclosureHeading as="h3" variant="heading30">
+    For product designers
+  </DisclosureHeading>
```

Comment:
I wonder if there are any natural groupings for these list items? One giant bulleted list feels a little overwhelming.

Reply:
Was hoping to discuss that. Any thoughts on how might we structure this content?

Reply:
First pass:

**Writing Text Content**

- Translate / spell out acronyms the first time you use them
- Label and describe the same things with the same terminology
- Ensure that content is written as clearly and simply as possible (Why it matters)
- Warn users of links that have unusual behaviors, like linking off-site, or loading a new tab
- Use textual labels to make voice activation cues obvious
- Make sure the purpose of a link is clearly described: "read more" vs. "read more about accessibility"

**Using Text**

- Make sure heading levels describe a logical section/subsection structure (Why it matters)
- Give all form elements permanently visible labels
- Give grouped form elements group labels
- Place labels above form elements
- Employ well-balanced, highly legible fonts (not too complex or elaborate)
- Do not use very thin font faces (Why it matters)
- Underline links — at least in body copy
- Avoid justified body text (Why it matters)
- Provide enough spacing between lines of text (`line-height`) (Why it matters)
- Avoid all-caps text (Why it matters)
- Do not mark up subheadings/straplines with separate heading elements
- Avoid images of text — text that cannot be translated, selected, or understood by assistive tech
- Begin long, multi-section documents with a table of contents
- Don't make users perform actions to reveal content unless completely necessary
- If content is meant to be hidden, ensure it is properly hidden to all users

**Using Graphical/Non-text Content**

- Provide alternatives and/or descriptions for complex visualizations
- Provide descriptive captions for figures
- Use well-established, therefore recognizable, icons and symbols

**Color & Animation**

- Include only clear, meaningful animations
- Do not rely on color as the only means of differentiation of visual elements
- Avoid pure white or pure black shades

**Interactions**

- Use the same design patterns to solve the same problems
- Make UI controls look like controls; give them strong perceived affordance
- Mark invalid fields clearly and provide associated error messages
- Indicate swipe gesture support clearly, and provide simple tap-based alternatives
- Avoid time constraints where possible; provide a clear warning and option to extend where not possible
- Do not instate "infinite scroll" by default; provide buttons to load more items
- Ensure primary calls to action are easy to recognize and reach
- Do not auto focus form fields, on page load
- Break up long and complex forms into discrete sections and/or screens
- Make forms as short as possible; offer shortcuts like autocompleting the address using the postcode
- Inform the user when there are important changes to the application state